core/templates/domain/topic/topic-rights-backend-api.service.ts

```diff
@@ -42,15 +43,15 @@ export class TopicRightsBackendApiService {
 
   private _fetchTopicRights(
       topicId: string,
-      successCallback: (value?: TopicRightsBackendResponse) => void,
+      successCallback: (value?: TopicRightsBackendDict) => void,
       errorCallback: (reason?: string) => void): void {
     let topicRightsUrl = this.urlInterpolationService.interpolateUrl(
       TopicDomainConstants.TOPIC_RIGHTS_URL_TEMPLATE, {
         topic_id: topicId
       });
 
     this.httpClient.get(topicRightsUrl).toPromise().then(
-      (response: TopicRightsBackendResponse) => {
+      (response: TopicRightsBackendDict) => {
```

Comment:
actual response from backend:
![Screenshot from 2021-05-21 16-46-36](https://user-images.githubusercontent.com/24855641/119138593-ad794f00-ba5f-11eb-8782-ad911fad586e.png)

interface TopicsRightsBackendResponse:
![image](https://user-images.githubusercontent.com/24855641/119138686-c2ee7900-ba5f-11eb-8f55-e7fc58b87e7c.png)

interface TopicsRightBackendDict:

![image](https://user-images.githubusercontent.com/24855641/119138735-d4378580-ba5f-11eb-9519-340cf1a23d00.png)

Or should I change the `TopicRightsBackendResponse` interface?

Reply:
TopicRightsBackendResponse is more suitable

Reply:
`TopicRightsBackendDict` seems accurate. Thanks!